controllers/apply/awards-for-all/form.js

```diff
@@ -745,6 +745,13 @@ module.exports = function({ locale, data = {} }) {
             }),
             isBilingual: true,
             allFields: fields,
+            featuredErrorsAllowList: [
```

Comment:
This list is a whitelist of fields we'll show by default on the summary screen. If `includeBaseError` is `true` then the error will be shown in all cases, if `false`, only when the field has a value (i.e. exclude the `base` message).